src/framework/components/element/text-element.js

```diff
@@ -1630,6 +1711,44 @@ Object.assign(pc, function () {
         }
     });
 
+    Object.defineProperty(TextElement.prototype, 'enableMarkup', {
+        get: function () {
+            return this._enableMarkup;
+        },
+        set: function (value) {
+            value = !!value;
+            if (this._enableMarkup === value) return;
+
+            this._enableMarkup = value;
+
+            if (this.font) {
+                this._updateText();
+            }
+        }
+    });
+
+    Object.defineProperty(TextElement.prototype, 'symbols', {
+        get: function () {
+            return this._symbols;
+        }
+    });
+
+    Object.defineProperty(TextElement.prototype, 'symbolColors', {
+        get: function () {
+            if (this._symbolColors === null) {
+                return null;
+            }
+            return this._symbolColors.map(function (c) {
+                return this._colorPalette.slice(c * 3, c * 3 + 3);
+            }, this);
+        }
+    });
+
+    Object.defineProperty(TextElement.prototype, 'rtl', {
+        get: function () {
+            return this._rtl;
```

Comment:
Is this not the same as this._rtlReorder? Maybe we can drop the `rtl` property if so?

Reply:
no they mean different things:
-rtlReorder flag indicates that the text should be checked for bidirectional text (and should have the bidi algorithm run on it)
-rtl flag indicates that the text has been checked and is right-to-left

Reply:
Does this `rtl` flag need to be a public property?

Reply:
I think it's useful... but it's not documented so I thought that was legit?

Reply:
Kk